Portal/src/Datahub.Portal/Pages/Workspace/Toolbox/WorkspaceToolboxPage.razor

```diff
@@ -105,45 +123,46 @@
 </MudStack>
 
 @code {
-
     [Parameter] public string WorkspaceAcronym { get; set; }
 
     private Datahub_Project _workspace;
 
     private readonly List<string> _toolList =
     [
         TerraformTemplate.AzureDatabricks,
-        TerraformTemplate.AzureStorageBlob,
-        TerraformTemplate.AzureAppService,
-        TerraformTemplate.AzurePostgres,
-        TerraformTemplate.AzureArcGis,
-        TerraformTemplate.AzureAPI
+    TerraformTemplate.AzureStorageBlob,
+    TerraformTemplate.AzureAppService,
+    TerraformTemplate.AzurePostgres,
+    TerraformTemplate.AzureArcGis,
+    TerraformTemplate.AzureAPI
     ];
 
-    private enum ToolStatus
+    private enum AvailabilityStatus
     {
         Available,
-        SendingRequest,
-        PendingRequest,
-        Disabled,
+        // SendingRequest,
+        // PendingRequest,
+        // Disabled,
+        // PendingDeletion,
         UnderDevelopment,
-        Exists,
-        MetadataRequired,
-        Error
+        // Exists,
+        MetadataRequired
+        // Error
     }
 
-    private readonly Dictionary<string, ToolStatus> _toolCurrentDevelopmentStatusMap = new()
+    private readonly Dictionary<string, ProjectResourceStatus> _toolCurrentDevelopmentStatusMap = new()
     {
-        { TerraformTemplate.AzureDatabricks, ToolStatus.Available },
-        { TerraformTemplate.AzureStorageBlob, ToolStatus.Available },
-        { TerraformTemplate.AzureAppService, ToolStatus.Available },
-        { TerraformTemplate.AzurePostgres, ToolStatus.Available },
-        { TerraformTemplate.AzureArcGis, ToolStatus.UnderDevelopment },
-        { TerraformTemplate.AzureAPI, ToolStatus.UnderDevelopment }
+        { TerraformTemplate.AzureDatabricks, ProjectResourceStatus.Available },
+        { TerraformTemplate.AzureStorageBlob, ProjectResourceStatus.Available },
+        { TerraformTemplate.AzureAppService, ProjectResourceStatus.Available },
+        { TerraformTemplate.AzurePostgres, ProjectResourceStatus.Available },
+        { TerraformTemplate.AzureArcGis, ProjectResourceStatus.UnderDevelopment },
+        { TerraformTemplate.AzureAPI, ProjectResourceStatus.UnderDevelopment }
```

Comment:
I was going to make these changes but then I realized it would be a dictionary of <string, AvailabilityStatus> which would cause issues with a lot of the rest of the code since it's comparing/using ProjectResourceStatus. i think the whole thing would need to be cleaned up/refactored to support this but I can def give it a shot if it's of particular importance!